docs/2-tutorials/2-develop/6-push-notifications.html.md

```diff
@@ -2,7 +2,164 @@
+
+### Sending Your First Push
+
+1. Open a new terminal window and enter the following command if you are testing on iOS:
+
+		$ phonegap push --deviceID APA91bE1MmeTc92igNoi5OkDWUV --service apns --payload '{ "aps": { "alert": "Hello World" } }'
+
+   or enter the following command if you are testing on Android:
+
+		$ phonegap push --deviceID APA91bE1MmeTc92igNoi5OkDWUV --service gcm --payload '{ "data": { "title": "Hello", "message": "World"} }'
+
+   replacing the deviceID with the one you received from the registration event in the previous step.
+
+2. This will result in a push notification being received by the app.
+
+   <img class="mobile-image" src="/images/dev-app-success-push.jpg" alt="Developer App, connection success"/>
+
+### Explaining Push
+
+Now open the **www/index.js** file and we'll take a deep dive into what is going on. All of the code required to register and receive push notifications is in the `onDeviceReady` function.
+
+#### Initialization
+
+First we need to initialize the push notification system. The `init` function sets up the communication path with the remote push service and requests a registration ID. You will need to do this each time your app starts. Don't worry about requesting a registration ID each time your app starts. The phonegap-plugin-push code is smart enough to use cached registration ID if available.
+
+		var push = PushNotification.init({
+				"android": {
+						"senderID": "XXXXXXXX"
+				},
+				"ios": {
+						"sound": true,
+						"vibration": true,
+						"badge": true
+				},
+				"windows": {}
+		});
+
+<div class="alert--info"> **NOTE:** You may notice the `senderID` is set to *XXXXXXXX*. That is okay as the **PhoneGap Developer App** will intercept this request and use the correct value. When you build an app for yourself you will need to get a `senderID` when you setup a project with Google to use GCM. </div>
+
+#### Registration
+
+Next we'll setup an event handler for the `registration` event. Once the app has successfully registered with the remote push service our event handler will be called. The function is called with only one parameter which is a data object and the object only has one property the `registrationId` received from the remote push service.
+
+Then we'll get the value of the saved `registrationId` from `localStorage`. If the old registration ID does not match the newly received registration ID we need to do two things. First, save the new `registrationId` into `localStorage`. Second, and the implementation is left up to the user, send the registration ID to your push service. For the purpose of this guide that functionality is not needed.
+
+You'll see that the `registration` event handler updates the HTML to visually display that the app is now registered. It does this by setting the CSS display attribute to none on the initial <p> element that was shown and instead shows the Registered element in index.html by setting its display attribute to block.
```

Comment:
You'll need to escape that HTML p element in this paragraph since it will probably use it literally - use &lt;p&gt;